### PR TITLE
Fix action required for pr 61373

### PR DESCRIPTION
**Pull request by msau42**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
As pointed out in https://github.com/kubernetes/kubernetes/issues/72257#issuecomment-449518727, the release note did not cover all impacted "from" versions

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.11.md [hunk 1]
```diff
@@ -2104,7 +2104,7 @@ filename | sha256 hash
 
 * NONE ([#62643](https://github.com/kubernetes/kubernetes/pull/62643), [@xiangpengzhao](https://github.com/xiangpengzhao))
 * ACTION REQUIRED: Alpha annotation for PersistentVolume node affinity has been removed.  Update your PersistentVolumes to use the beta PersistentVolume.nodeAffinity field before upgrading to this release ([#61816](https://github.com/kubernetes/kubernetes/pull/61816), [@wackxu](https://github.com/wackxu))
-* ACTION REQUIRED: In-place node upgrades to this release from versions 1.7.14, 1.8.9, and 1.9.4 are not supported if using subpath volumes with PVCs.  Such pods should be drained from the node first. ([#61373](https://github.com/kubernetes/kubernetes/pull/61373), [@msau42](https://github.com/msau42))
+* ACTION REQUIRED: In-place node upgrades to this release from at least versions 1.7.14, 1.8.9, and 1.9.4 are not supported if using subpath volumes with PVCs.  Such pods should be drained from the node first. ([#61373](https://github.com/kubernetes/kubernetes/pull/61373), [@msau42](https://github.com/msau42))
```
Comment (liggitt) [inline CHANGELOG-1.11.md]: maybe rephrase as 1.7.14+, 1.8.9+, and 1.9.4+?

Comment (msau42) [reply CHANGELOG-1.11.md]: done
